lib/ios-crash-log.js

```diff
@@ -0,0 +1,63 @@
+import path from 'path';
+
+import _ from 'lodash';
+import _glob from 'glob';
+import { fs, getLogger } from 'appium-support';
+import B from 'bluebird';
+let glob = B.promisify(_glob);
+
+
+require('date-utils');
+let logger = getLogger('IOS-crash-log');
+
+class IOSCrashLog { 
+  constructor() {
+    this.prevLogs = [];
+    this.logsSinceLastRequest = [];
+    this.crashDir = path.resolve(process.env.HOME, 'Library', 'Logs', 'DiagnosticReports');
+  }
+
+  async getCrashes () {
+    try {
+      return glob(`${this.crashDir} /*.crash`);
```

Comment:
this is kind of a tricky case. the `try/catch` will only work if we actually `await`, so in this case, it does really need to be `return await glob(...);`